src/hb-open-type.hh

```diff
@@ -724,9 +724,16 @@ struct SortedArrayOf : ArrayOf<Type, LenType>
   template <typename SearchType>
   inline int bsearch (const SearchType &x) const
   {
+    int max = (int) this->len - 1;
+    /* Special case, should be between 0.7% to 3% while formatting text. */
+    if (max == 0) {
+      return this->arrayZ[0].cmp (x);
+    }
```

Comment:
No brackets needed here.  Also, since we are micro-optimizing, maybe move it earlier and just check this->len == 1?